### DISCUSSION TITLE
[Question][Hw5] is the title of Hw5, 'cs492-2023-fall-hw5-HLe4s-kaist' right?

### DISCUSSION THREAD

**Discussion by HLe4s-kaist** [prosyslab-classroom/cs524-program-analysis#305] [Q&A]
Name: ChanU Kang

YOUR QUESTION:

EN 
I accepted dispatched, new homework, hw5. but the name of the hw5 is 'cs492-2023-fall-hw5-HLe4s-kaist' ...
Should I reject the homework and wait to correct it?
or.. 
The name is not matter. so I can just go and do my homework already accepted'?
----
KR (I'm not sure to make myself clear in English, So I'd rather write down some korean summary)
HW5가 나왔다고 해서, 미리 푸려고 github assingment를 accept해서 레포지토리가 생겼는데
이름이 'cs492-2023-fall-hw5-HLe4s-kaist' 인데... 그냥 제목만 다른거고, 내용은 같으니까. 그냥 숙제를 하면 되는건가요?
아니면 다시 업데이트를 해주시길 기다려야 하나요??

노고에 감사드립니다.

강찬우 올림.

**Comment by goodtaeeun** [comment]
The link was incorrect. Sorry for the inconvenience. I will update it right away.

**Comment by HLe4s-kaist** [comment]
Thank you! when the link is updated, I will re-accept it.

**Comment by goodtaeeun** [comment]
The link was actually correct but just the name was wrong. You can delete the current repository and regenerate a new one.
